src/components/PostPageDetail/index.js

```diff
@@ -22,7 +30,7 @@ import {
   ANALYTICS_SHARE_POST_PDP_SCREEN,
   SOURCE_PDP
 } from '../../utils/constants';
-import {fonts} from '../../utils/fonts';
+import {fonts, normalize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `normalize` function has been added to the import statement from `../../utils/fonts`. Ensure that this function is being used in the code and that it exists in the imported module.
```diff
- import {fonts} from '../../utils/fonts';
+ import {fonts, normalize} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->